docs/dev/engineering/overview.md

```diff
@@ -0,0 +1,5 @@
+# Engineering
+
+This section details how to get your hands dirty and contribute to the codebase.
+
+Use the navigation to the left to continue.
```

Comment:
`Navigation to the left` on github repo shows nothing.

Reply:
Yes, indeed, this documentation is meant to be consumed rendered.